tests/fakeprotocol_smartcam.py

```diff
@@ -34,6 +34,7 @@ def __init__(
         list_return_size=10,
         is_child=False,
         verbatim=False,
+        components_not_included=False,
```

Comment:
It is not directly clear what's the reasoning for this, so I'd add a quick and dirty docstring, or a comment below where this is used.

Reply:
It's useful for testing lists but have removed it for now.